### DISCUSSION TITLE
Teleport session recording to AWS S3

### DISCUSSION THREAD

**Discussion by AryanSemwal** [gravitational/teleport#7456] [Q&A] [answered]
How can I configure my teleport to send the session recordings to the AWS S3 bucket.

**Comment by webvictim** [comment, marked as answer]
See our docs: https://goteleport.com/docs/admin-guide/#using-amazon-s3

There are also details on configuring IAM policies for S3 with Teleport here: https://goteleport.com/docs/aws-oss-guide/#iam-for-amazon-s3
